examples/pygeosxExamples/hydaulicFractureWithMonitor/hydraulicFractureWithMonitor.py

```diff
@@ -0,0 +1,57 @@
+
+import sys
+from mpi4py import MPI
+import pygeosx
+from pygeosx_tools import wrapper, xml
+import matplotlib.pyplot as plt
+
+
+# Get the MPI rank
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+
+
+def run_problem():
+    """
+    Run the GEOSX problem
+    """
+    # Initialize the code
+    xml.apply_xml_preprocessor()
+    problem = pygeosx.initialize(rank, sys.argv)
```

Comment:
Please provide `sys.argv` as input argument of the function to reduce side effects and increase testability.